repos/system_upgrade/common/actors/opensshconfigscanner/libraries/readopensshconfig.py

```diff
@@ -12,14 +15,23 @@ def line_empty(line):
     return len(line) == 0 or line.startswith('\n') or line.startswith('#')
 
 
-def parse_config(config):
+def parse_config(config, ret=None, depth=0):
     """Parse OpenSSH server configuration or the output of sshd test option."""
```

Comment:
This requires a better argument names and their docstrings.

```suggestion
def parse_config(config, default_return_val=None, current_cfg_depth=0):
    """
    Parse OpenSSH server configuration or the output of sshd test option.
    
    :param Optional[OpenSshConfig] default_return_val: Default return value returned in case <?>
    :param int current_cfg_depth: How many recursive config includes were already followed. 

    """
```

Reply:
Thank you for the suggestions! I would be for improving the naming of `ret` to `base_config` as this matches more what it is used for (extending previously created configuration structure).

Regarding to the `depth`, the description sounds good, but it should be only internal. Is there a good way to express this in the python/docstring? Start wit underscore?